include/z64player.h

```diff
@@ -690,7 +690,7 @@ typedef struct WeaponInfo {
 #define PLAYER_STATE2_6 (1 << 6)
 #define PLAYER_STATE2_7 (1 << 7)
 #define PLAYER_STATE2_8 (1 << 8)
-#define PLAYER_STATE2_FORCE_SAND_FLOOR_SOUND (1 << 9)
+#define PLAYER_STATE2_FORCE_SAND_FLOOR_SOUND (1 << 9) // Forces dirt footstep sounds regardless of current floor type
```

Comment:
big +1 for the comments on the flags <3 is this one accurate though? flag says "force sand", comment says "force dirt", which is it?

Reply:
The sfx offset is currently called `SURFACE_SFX_OFFSET_SAND` but the one use of the flag in the game is on the dirt in front of graves lol. I cannot imagine in my head what the difference even i between dirt and sand sounds in the game.

Anyway ill change it

Reply:
lol yea I just double checked and the sound behind the graves is the same as the sound in haunted wasteland, so sand is definitely the right one. just think of it as grave sand instead of grave dirt, problem solved!